docs/en/changelog.md

```diff
@@ -1,5 +1,43 @@
 ## Changelog
 
+### v1.0.0rc2 (1/5/2022)
+
+#### Highlights
+
+- Support spconv 2.0
+- Support MinkowskiEngine with MinkResNet
+- Support training models on custom datasets with only point clouds
+- Update Registry to distinguish the scope of built functions
+- Replace mmcv.iou3d with a set of bird-eye-view (BEV) ops
+
+#### New Features
+
+- Add loader arguments in the configuration files (#1388)
+- Support spconv 2.0 with a solution for compatibility with cuda 9.0 (#1421)
```

Comment:
We can explain more about it, e.g.: 
- Support [spconv 2.0](https://github.com/traveller59/spconv) when the package is installed. Users can still use spconv 1.x in MMCV with CUDA 9.0 (only cost more memory) without losing the compatibility of model weights between two versions (#1421)